external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -282,6 +282,9 @@ class TrainConfig(TransformedParameters):
         nfiles: Number of files to use from train_url
         nfiles_valid: Number of files to use from test_url
         log_level: what logging level to use
+        save_only: If true, don't train, but save the train and test data with
+            tf.data.experimental.save to ``out_url``
+        data_format: one of ['nc', 'tf']
```

Comment:
Can you describe what this option does, and remove "netcdfs" from the train/test_url descriptions above?